crates/wasmparser/src/readers/code_section.rs

```diff
@@ -25,6 +25,10 @@ pub struct FunctionBody<'a> {
     allow_memarg64: bool,
 }
 
+impl<'a> Copy for FunctionBody<'a>{
+
+}
+
```

Comment:
This should be `#[derive(Copy)]` instead of a handwritten impl.

Reply:
Gotcha